share/picongpu/benchmarks/SPEC/include/picongpu/param/radiation.param

```diff
@@ -0,0 +1,205 @@
+/* Copyright 2013-2021 Rene Widera, Richard Pausch
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+#ifndef PARAM_RADFORMFACTOR
+#    define PARAM_RADFORMFACTOR radFormFactor_Gauss_spherical
+#endif
+
+#ifndef PARAM_RADWINDOWFUNCTION
+#    define PARAM_RADWINDOWFUNCTION radWindowFunctionTriplett
+#endif
+
+
+/*
+  radiation verbose level:
+  0=nothing, 1=physics, 2=simulation_state, 4=memory, 8=critical
+*/
+
+#define PIC_VERBOSE_RADIATION 3
+
+#include "picongpu/algorithms/Gamma.def"
+#include "picongpu/particles/manipulators/manipulators.def"
+#include "picongpu/plugins/radiation/VectorTypes.hpp"
+#include "picongpu/plugins/radiation/debug/PIConGPUVerboseLogRadiation.hpp"
+#include "picongpu/traits/attribute/GetMass.hpp"
+
+namespace picongpu
+{
+    namespace plugins
+    {
+        namespace radiation
+        {
+            namespace linear_frequencies
+            {
+                namespace SI
+                {
+                    constexpr float_64 omega_min = 0.0;
+                    constexpr float_64 omega_max = 1.06e16;
+                } // namespace SI
+
+                constexpr unsigned int N_omega = 1024; // number of frequencies
+            } // namespace linear_frequencies
+
+            namespace log_frequencies
+            {
+                namespace SI
+                {
+                    // plasma omega = sqrt( (electron density * (1.6e-19)^2) / (8.854e-12 * 9.11e-31) )
+                    //              = 1.78e14 1/s
+                    constexpr float_64 omega_pe = 1.78e14;
+                    constexpr float_64 omega_min = 0.1 * omega_pe;
+                    constexpr float_64 omega_max = 200 * omega_pe;
+                } // namespace SI
+
+                constexpr unsigned int N_omega = 256; // number of frequencies
```

Comment:
discussed offline, we should increase the number of frequencies to 512 and check that the ranges will not be cut by the newquest frequency.